apps/marketplace/components/SellerNotify/SellerNotifyIntroduction.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import Button from '@gov.au/buttons/lib/js/react.js'
+
+export class SellerNotifyIntroduction extends Component {
+  constructor(props) {
+    super(props)
+    this.handleContinueClick = this.handleContinueClick.bind(this)
+  }
+
+  componentDidMount() {
+    this.props.setStageStatus('introduction', 'doing')
+    this.props.setStageDoneStatus('introduction', true)
+  }
+
+  handleContinueClick(e) {
+    e.preventDefault()
+    this.props.setStageStatus('introduction', 'done')
+    this.props.setStageDoneStatus('introduction', true)
+    this.props.moveToNextStage('introduction')
+  }
+
+  render() {
+    if (this.props.flow === 'unsuccessful') {
```

Comment:
was `flow` really required for first version?  (not saying we remove it, just feels premature)

Reply:
It is premature, but I made the page and components to be generic enough that future flows can be added with ease given we are expecting a few, vs having to refactor for it when they come.